recipes/proj-data/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0" %}
+
+package:
+  name: proj-data
+  version: {{ version }}
+
+source:
+  url: http://download.osgeo.org/proj/proj-data-{{ version }}.tar.gz
+  sha256: 2637183bb2fa6cd421085734e5212b0c12550b8d5d3ecaadbfc1044648c7a906
+  folder: data
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - proj >=7.0.0
+
+test:
+  commands:
+    - test -f $PREFIX/share/proj/ca_nrc_ntv1_can.tif
+    - echo -111 50 | cs2cs +proj=latlong +ellps=clrk66 +nadgrids=ca_nrc_ntv1_can.tif +to +proj=latlong +ellps=GRS80 +datum=NAD83
+    - echo -105 40 | cs2cs +init=epsg:4326 +to +init=epsg:2975
+
+about:
+  home: https://github.com/OSGeo/proj-data
+  license: Public domain | X/MIT | BSD-2/3/4-Clause | CC0 | CC-BY (v3.0 or later) | CC-BY-SA (v3.0 or later)
+  # Each grid has a *README.txt file with the license info, that is copied with
```

Comment:
Just leave this comment here. We don't need to list all the licenses below b/c that would be a huge mess.

Reply:
Sure, but if it is licensed under CC0, that is all we need to add.  We must have *some* license

Reply:
The licenses ARE in the package in the authors designed directories. We DO have all of them. This is does not copy the files over to the conda directory and that is a requirement that we created to cover the many cases where the file(s) did not even existed.

Reply:
`licesne_file` can be a list.  What is the problem with including at least one of them?

Reply:
Each data set has its own license.  It is not that all these licenses apply to all the data.  If we picked an arbitrary data license for an arbitrary data set, this would give the false impression that this license applied to all the data sets.

Reply:
It will be misleading and wrong. Also, while the distribution of those data with multiple licenses is fine, listing a license or some licenses without mentioned which data they belong we will be infringing some of the license.